### PR TITLE
CSI drivers as WASI modules

### PR DESCRIPTION
**Pull request by mhmxs**
This change replaces the hard-coded, compile-time version of the AWS EBS driver with a dynamically loaded WebAssembly (WASI) module. The CSI driver has a clear interface and a nice example in the repo.

This change makes really easy to write any CSI integration for the project in any language. New CSI driver implementation has to read environment variables and write the result to the standard output. This is the simplest integration possible.

The new driver needs to be mounted at the operator pod's `/directory/[STORAGECLASS_PROVIDER_NAME]` directory, and the operator would automatically load them during startup time.

### CODE REVIEW

**Review by rovandep — COMMENTED** [review]
fantastic work! went to the changes and will try to the images later today or tomorrow then approve the PR.

**Comment by mhmxs** [comment]
@rovandep real e2e testing of my side also missing, would like to do it on Monday.

File: pkg/drivers/drivers.go [hunk 1]
```diff
@@ -1,25 +1,223 @@
...
+
+	for _, file := range files {
+		if !file.IsDir() {
+			continue
+		}
+
+		driverPath := fmt.Sprintf("%s/%s/main.wasm", DriversDir, file.Name())
+
+		if _, err := os.Stat(driverPath); err != nil {
+			log.Printf("unable to found main.wasm for %s: %s", file.Name(), err.Error())
```
Comment (nolancon) [inline pkg/drivers/drivers.go:33]: nit: `unable to find`

**Review by nolancon — DISMISSED** [review]
I have very little context here, but I can't see anything wrong so 🟢 from me so you are not blocked.

**Review by nolancon — APPROVED** [review]

**Pull request merged**